rabbitmq/README.md

```diff
@@ -20,9 +20,27 @@ The RabbitMQ check is included in the [Datadog Agent][3] package. No additional
 
 ### Configuration
 
+Rabbitmq exposes metrics in two ways: the [RabbitMQ Management Plugin][4] and the [Rabbitmq Prometheus Plugin][19]. The RabbitMQ integration supports both of these plugins.
+
 #### Prepare RabbitMQ
 
-Enable the [RabbitMQ Management Plugin][4]. The Agent user then needs at least the `monitoring` tag and these required permissions:
+##### [RabbitMQ Prometheus Plugin][19].
```

Comment:
We're only using OMv2 for the prometheus setup right? Do you think we should leave a disclaimer that this is only available for python3?

Reply:
Good point, I'll add that